WordPressKit.xcodeproj/project.pbxproj

```diff
@@ -3519,7 +3516,6 @@
 				DYLIB_INSTALL_NAME_BASE = "@rpath";
 				INFOPLIST_FILE = WordPressKit/Info.plist;
 				INSTALL_PATH = "$(LOCAL_LIBRARY_DIR)/Frameworks";
-				IPHONEOS_DEPLOYMENT_TARGET = 11.0;
```

Comment:
It's not clear in this diff, but by opening the project in Xcode I saw you decided to remove the target-level deployment setting, making them all use the project-level one.

Great idea 👍